provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.1-node

```diff
@@ -36,7 +36,9 @@
         }
       },
       "s3_instances": "TMPL_S3SERVER_INSTANCES_COUNT",
+      "storage_set_id": "TMPL_STORAGE_SET_ID",
       "storage": {
+        "cvg_count": "TMPL_CVG_COUNT", 
```

Comment:
As we are updating this file, I would suggest to update all other template and sample files with similar changes.
Thanks!

Reply:
Agreed. Can we take it as a separate task? This PR was created as RE was stuck in miniprovisioner job for CentOS7.9 and this helped.